### ISSUE TITLE
Inquire about the general training duration.

### ISSUE THREAD

**Issue by wangshansong1** [facebookresearch/sam3#189]
Hi, I'm trying to fine-tune my data and I'm having some issues.

1. I added a "checkpoint_path" key under "trainer.model" in my YAML file and filled in the value: "path/to/sam3.pt". This is my first step in trying to fine-tune based on sam3. There were no errors, but there was no message in the logs indicating that it was loaded successfully. So is this the correct step?
2. On 8 B200 GPUs, the maximum batch size per GPU is set to 1. Increasing it to 2 causes an OutOfMemoryError (OOM), and the time per batch is 1.88s. Are these phenomena normal during training?

By the way, should line 143 of "roboflow_v100_full_ft_100_images.yaml" be corrected to "sam3.train.loss.loss_fns.SemanticSegCriterion"? It might be a minor bug.

Best

**Comment by 4ndr3aR** [comment]
Yes, "sam3.train.loss.loss_fns.SemanticSegCriterion" is the correct loss to call, but I'm still getting CUDA OOMs even with bs=1 on a RTX A6000 (45 Gb of VRAM). Did you manage to reduce memory consumption somehow?
